From 9a7d1c307a7f85f90e10d7c7a1f854bb16fe7723 Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Tue, 30 Jul 2024 09:44:28 +0200
Subject: [PATCH] Update Makefile

---
 src/_ext_src/Makefile | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/_ext_src/Makefile b/src/_ext_src/Makefile
index 3e132cd7..cbeda033 100644
--- a/src/_ext_src/Makefile
+++ b/src/_ext_src/Makefile
@@ -8,6 +8,9 @@ OMP_F90_FLAGS=
 OMP_F2PY_FLAGS=
 
 # compiler specific variables
+conda: F90=$(F90)
+conda: F90FLAGS=$(FFLAGS)
+
 ifort: F90=ifort
 ifort: F90FLAGS=-fpp -no-prec-div -fast-transcendentals
 ifort: ARCH=-xHost
